### PR TITLE
[ETD-145] Update for newer modules versions

### PR DESCRIPTION
**Pull request by Ficzer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-# Remote-Target-Runner
\ No newline at end of file
+# Remote-Target-Runner
```
Comment (Lurkerpas) [inline README.md:1]: Please provide https://learnpython.com/blog/python-requirements-file/

Comment (Ficzer) [reply README.md:1]: Done

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,141 @@
+antlr==2.7.5rc1
+antlr-python3-runtime @ file:///home/taste/tool-src/antlr/antlr3_python3_runtime_3.4
```
Comment (Lurkerpas) [inline requirements.txt]: Do we need hardcoded absolute paths? And do we actually need ANTLR  and ASN.1 value editor for remote target runner? I think that some dependencies are superfluous....

Comment (Ficzer) [reply requirements.txt]: Removed unnecessary requirements

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**